systemDesign/yt-startSeconds.md

```diff
@@ -0,0 +1,27 @@
+# YouTube Player API Reference
+
+The IFrame player API lets you embed a YouTube video player on your website and control the player using JavaScript.
+
+Using the API's JavaScript functions, you can queue videos for playback; play, pause, or stop those videos; adjust the player volume; or retrieve information about the video being played. You can also add event listeners that will execute in response to certain player events, such as a player state change.
+
+## startSeconds
+
+The property <b>startSeconds</b> lets you skip the given seconds from the start.
```

Comment:
Please use the markdown syntax: https://www.markdownguide.org/basic-syntax/#code https://www.markdownguide.org/basic-syntax/#bold